Document StopFake5 (#65)
ambanum/disinfo.quaidorsay.fr: _pages/en/encyclopedia/3-qualification/4-actors.md

This is a factchecking and debunking platform for all of the French broadcasting audience. It gathers content produced by various media such as Arte, France Médias Monde, France Télévisions, Radio France and TV5 Monde.

### In Ukraine

> #### [StopFake5](https://www.stopfake.org/) 

Reviewer: ```suggestion
#### [StopFake](https://www.stopfake.org/) 
```
The '5' doesn't seem relevant anymore.

> [Stop Fake 5](https://www.stopfake.org/en/about-us/) is a fact-checking website focusing on fake informations regarding Ukraine, most of which involve Russia. 

Reviewer: ```suggestion
[Stop Fake](https://www.stopfake.org/en/about-us/) is a fact-checking website focusing on fake informations regarding Ukraine, most of which involve Russia. 
```

It was launched in 2014, around the time of the annexation of Crimea, by the Kyiv Mohyla Journalism School and the "Digital Future of Journalism" project of the [Kyiv Mohyla Academy](https://www.ukma.edu.ua/eng/).

> Fostered by journalists and IT people, among others, the website fact-cheks disinformation and propaganda about events in Ukraine, but also more widely provides analyses on "Kremlin propaganda", in the shape of video digests, radio podcasts and articles.

Reviewer: ```suggestion
Fostered by journalists and IT people, among others, the website fact-checks disinformation and propaganda about events in Ukraine, but also more widely provides analyses on content they consider to be "Kremlin propaganda", in the shape of video digests, radio podcasts and articles.
```
